### ISSUE TITLE
[Boards] Agile process: Resolved state should be in resolved category for User Story work item type

### ISSUE THREAD

**Issue by NenoLoje** [NenoLoje/AzureDevOpsWishlist#7]
### Current state

In the Agile process template:

- For a Bug, the `Resolved` state is in the *Resolved* category.
- For a User Story, the `Resolved` state is in the *In Progress* category.

### Expected state

This would be consistent among the work item types. The `Resolved`state should be in the *Resolved* category.

#### User Story work item

> 
![grafik](https://user-images.githubusercontent.com/13981145/189182863-7ae2edd8-8262-418b-beeb-4a1e9fd8ba4f.png)

#### Bug work item

> 
![grafik](https://user-images.githubusercontent.com/13981145/189182921-9ad43472-6271-45fb-a15e-a38325c1477b.png)

**Comment by danhellem** [comment]
yeah been like this since day 1. Not sure why though. We have it on our list but there is worry changing it would have unintended consequences. No current plans to do anything about it yet. but I am tracking in.
